### DISCUSSION TITLE
Can I "assemble" a full quarto doc with all includes included?

### DISCUSSION THREAD

**Discussion by octomike** [quarto-dev/quarto-cli#11267] [Q&A]
### Description

For better structuring I refactored a larger report using multiple `{{< include ... >}}` blocks.

However, now I am unable to actually debug the document, because there is no apparent way to run code chunks inside the included blocks from the main document (I use Rstudio).

As a workaround, I tried to figure out if there is a way to "assemble" all includes, creating a "full" quarto document. Then, I would be able to step through the chunks again.  I assume the shortcode is doing this under the hood, but I can't find anything exposing this functionality to me.

Am I doing this wrong?

**Comment by cscheid** [comment]
There's no fully-automatic way to do this in Quarto, but `quarto inspect` gives you the set of include statements (and their include statements, transitively) so you can put the pieces together.
